web/satellite/src/components/apiKeys/ApiKeysArea.vue

```diff
@@ -2,54 +2,138 @@
 // See LICENSE for copying information.
 
 <template>
-    <div>
-        <div v-if="!isEmpty" class="api-keys-header">
-            <HeaderArea/>
-        </div>
-        <div v-if="!isEmpty" class="api-keys-container">
-            <div class="api-keys-container__content">
-                <div v-for="apiKey in apiKeyList" v-on:click="toggleSelection(apiKey.id)">
-                    <ApiKeysItem
-                        v-bind:class="[apiKey.isSelected ? 'selected': null]"
-                        :apiKey="apiKey" />
+    <div class="api-keys-area">
+        <h1>Api Key</h1>
+        <div class="api-keys-area__container">
+            <div class="api-keys-area__container__new-api-key" v-if="isNewApiKeyPopupShown">
```

Comment:
We should consider creating separate component for new api key creation to make ApiKeysArea smaller and moreover simplier